setup.sh

```diff
@@ -56,27 +57,42 @@ echo "Edit ./configs/config.json to modify any other config."
 
 function Pokebotinstall () {
 cd $pokebotpath
-if [ -f /etc/debian_version ]
+if [ "$(uname -s)" == "Darwin" ]
+then
+echo "You are on Mac os"
+sudo brew update 
+sudo brew install --devel protobuf
+elif [ -x "$(command -v apt-get)" ]
 then
 echo "You are on Debian/Ubuntu"
 sudo apt-get update
-sudo apt-get -y install python python-pip python-dev build-essential git virtualenv 
-elif [ -f /etc/redhat-release ]
+sudo apt-get -y install python python-pip python-dev gcc make git
+elif [ -x "$(command -v yum)" ]
 then
 echo "You are on CentOS/RedHat"
-sudo yum -y install epel-release
-sudo yum -y install python-pip
-elif [ "$(uname -s)" == "Darwin" ]
+sudo yum -y install epel-release gcc make
+sudo yum -y install python-pip python-devel
```

Comment:
@nivong after a quick search, "python-devel" should be correct: http://stackoverflow.com/questions/23541205/cant-install-python-dev-on-centos-6-5